models/Trip.js

```diff
@@ -0,0 +1,10 @@
+import mongoose from "mongoose";
+const { Schema } = mongoose;
+
+const tripSchema = new Schema({
+  country: { type: String, required: true },
+});
+
+const Trip = mongoose.models.Trip || mongoose.model("Trip", tripSchema, "trips");
+
+export default Trip;
```

Comment:
I think about your data-structure: Is it necessary to have three collections? If a 'todo' is not used anywhere else it could be a good idea to have an array of 'todos' inside the 'destination'-collection instead of an own collection. 
The same for trips and destinations.

Reply:
I chose to separate the collections to keep it simple. A 3 layers nested document would cause a long nested object iteration and makes it challenging to change specific data.